oaknational/Oak-Web-Application
# Screenshots of unit modals added to curric automate script

## Description
Added optional screenshots of unit modals to the curriculum automation script. 

## Issue(s)

Fixes `CUR-1058`

## How to test
Note this will take a while to run, as it's screenshotting every sequence/unit-modal

1. Clone the repo
2. Start the dev server `npm run dev`
3. `./scripts/dev/curriculum/automate screenshot :foo --modals`
4. `./scripts/dev/curriculum/automate screenshot :bar --modals`
5. `./scripts/dev/curriculum/automate screenshot compare :foo :bar`

Check the page launched from `compare` command.

**Note:** It doesn't currently render the unit options only the unit options overview page currently.
